src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-size: 16px;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: bisque;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+h1 {
+  width: 100%;
+  font-size: 16px;
+  align-self: flex-start;
+  font-weight: 500;
+  padding: 1em;
+  border-bottom: 1px solid lightgrey;
+  margin: 0;
+}
+
+#listcontainer {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  box-shadow: 3px 3px 3px lightgrey, -3px 3px 3px lightgrey, 3px 3px 3px lightgrey;
+  width: 60%;
+  align-items: center;
+  max-height: 95vh;
+}
+
+.field {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
In order to follow DRY best practices, you should set a selector for the code that is being repeated, that will make the code much cleaner and allow to edit it better, the line setting the width to 100% is mentioned in the following lines:
1. https://github.com/asdt560/Review-repo/blob/5c9598f41427b83c02de69d2e0bc770fc8c41913/src/style.css#L14
2. https://github.com/asdt560/Review-repo/blob/5c9598f41427b83c02de69d2e0bc770fc8c41913/src/style.css#L36
3. https://github.com/asdt560/Review-repo/blob/5c9598f41427b83c02de69d2e0bc770fc8c41913/src/style.css#L48
4. https://github.com/asdt560/Review-repo/blob/5c9598f41427b83c02de69d2e0bc770fc8c41913/src/style.css#L87
5. https://github.com/asdt560/Review-repo/blob/5c9598f41427b83c02de69d2e0bc770fc8c41913/src/style.css#L93

Reply:
Yeah, I really needed to get around to fixing that, some of those aren't even needed.